speech-api/SpeechSynthesis-speak-twice.html

```diff
@@ -1,17 +1,22 @@
 <!doctype html>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<p>Some body text.
```

Comment:
You probably don't need this text now that you're using `test_driver.bless` instead. Ditto in following tests.

Reply:
Actually we still need it, since bless requires the document body to be around.

Reply:
Ah, if that's all you need then `<body>` would be clearer. Plenty of tests to that ahead of doing something with `document.body`.